sdk/search/Azure.Search.Documents/tests/Samples/Sample05_IndexingDocuments.cs

```diff
@@ -121,6 +121,13 @@ public async Task SimpleIndexing()
 
                 await WaitForDocumentCountAsync(searchClient, 1000);
 
+                // When using the free SKU, there may be enough load to prevent
+                // immediately replication to all replicas and we get back the
+                // wrong count. Wait another second before checking again. We
+                // may also upgrade to a basic SKU, but that will take longer
+                // to provision.
+                await DelayAsync(TimeSpan.FromSeconds(1));
```

Comment:
Thanks for looking at this, but why are we adding a delay after the `WaitForDocumentCountAsync` above?  That also has parameters that will let you stretch the number of times it checks and the amount it waits between checks if you want extra delay.  I believe it does 3 attempts today with a second between each.  We could bump that to two seconds, etc., if it's not been reliable enough in practice.

Reply:
The comment states that replicas need time to get updates. This is what @brjohnstmsft recommends. Given more test failures last night with this same problem, I'm moving this to `WaitForDocumentCountAsync` or we're just playing whack-a-mole. If you want more information, let's chat offline.